usagov/usagov-2021
# Usagov 2053 ceo error inline message

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2053

## Description

Added text "Error: " to inline error text in the CEO form
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
Goto http://localhost/elected-officials for English and http://localhost/es/funcionarios-electos
- Submit the form without filling it out so you get the error text
- Verify the inline error text has "Error: " before each inline error text
<img width="1512" alt="Screenshot 2024-11-06 at 10 45 37 AM" src="https://github.com/user-attachments/assets/243450bf-0e40-4657-b36b-5d9000086d15">

<img width="1512" alt="Screenshot 2024-11-06 at 10 45 49 AM" src="https://github.com/user-attachments/assets/24551013-56b1-43a5-994f-afc761f80659">

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
